src/examples/bus-journey-patterns/main.js

```diff
@@ -37,5 +37,6 @@ function tableRowHtml (atcocodeToStopName, journeyPatterns, rowIndex) {
 }
 
 function tableCellHtml (atcocodeToStopName, journeyPattern, rowIndex) {
-  return `<td><b>${atcocodeToStopName[journeyPattern.stops[rowIndex]?.atcocode] || ''}</b> ${journeyPattern.stops[rowIndex]?.atcocode || ''}</td>`
```

Comment:
What's the purpose of this split? Readibility, linter god?

Reply:
We were using safety navigation operator (`?.`) which is not supported by the version of node that Codesandbox use and the rendering of the example on their page was broken.

Reply:
Yes, I was asking exactly because I saw it's still used even after the split.